extensions/blocks/podcast-player/components/header.js

```diff
@@ -3,11 +3,37 @@
  */
 import { memo } from '@wordpress/element';
 
-const Header = memo( ( { track, children } ) => (
-	<div>
-		{ /* TODO: display info based on `track` */ }
-		{ track ? <div>{ track.title }</div> : null }
-		<div>{ children }</div>
+const Header = memo( ( { playerId, track, children } ) => (
+	<div className="jetpack-podcast-player__header-wrapper">
+		<div className="jetpack-podcast-player__header" aria-live="polite">
+			<div className="jetpack-podcast-player__track-image-wrapper">
+				<img
+					className="jetpack-podcast-player__track-image"
+					alt="Placeholder alt text"
+					src="https://placekitten.com/200/200"
+				/>
+			</div>
+
+			{ track && track.title ? (
+				<h3 id={ `${ playerId }__track-title` } className="jetpack-podcast-player__track-title">
+					{ track.title }
+				</h3>
+			) : null }
+		</div>
+
+		{ /* putting this above the audio player for source order HTML with screen readers, then visually switching it with the audio player via flex */ }
+		{ track && track.description ? (
+			<div
+				id={ `${ playerId }__track-description` }
+				className="jetpack-podcast-player__track-description"
+			>
+				This is the track description that will need to be escaped and rendered via HTML or the
+				like, since that's how the RSS feed delivers it.
+			</div>
+		) : null }
+
+		{ /* children contains the audio player */ }
```

Comment:
@marekhrabe This is the comment I needed 🙌 I looked at the Header file previously and was like, oh, there's no `children` being passed via the original function. I guess we don't need that here. Then I was looking up how to do Vue-style `slots` in React. This is how you do it 🤦‍♂😅

Reply:
No worries! I have not used them this way before but I think it works well here